src/frontend/src/handler/query.rs

```diff
@@ -69,10 +72,15 @@ pub async fn handle_query(
     };
 
     let rows_count = match stmt_type {
-        StatementType::SELECT => rows.len() as i32,
+        StatementType::SELECT => 0_i32,
```

Comment:
Why this is 0?

Reply:
for query statement, rows_count is meaningless because it's stream mode. We will calculate the row count when we receive the data. 
I have added the related comment in the rows_count field of PgResponse. 
https://github.com/risingwavelabs/risingwave/blob/37ce306acb54bf9c530ad5c5116138f218f57075/src/utils/pgwire/src/pg_response.rs#L80

Reply:
It would be better if we use `Option` here.